### ISSUE TITLE
Logo Proposal

### ISSUE THREAD

**Issue by Tobaloidee** [luizdepra/hugo-coder#25]
Hi Sir @luizdepra , I am a graphic designer and i want to contribute to your nice project by proposing a logo design if you will permit me. I think it will make the project more visible if it has a logo. If you allow me i will start designing and show it here. It will be my gift to the project for free.

thanks and best regards!

-Tobaloidee

**Comment by luizdepra** [comment]
Hey, @Tobaloidee.

Sure, why not? I really would appreciate it.

**Comment by Tobaloidee** [comment]
Okay Sir..ill show you here my proposal as soon as i finished it! Thanks thanks!

**Comment by Tobaloidee** [comment]
H in a negative space inside the coding symbol with simple and minimal color appearance

logomark
![hugo-coder-01](https://user-images.githubusercontent.com/36028424/40570871-1855d9ba-60c2-11e8-8c5f-db8ec6dca7f5.png)

logotypes
![hugo-coder-02](https://user-images.githubusercontent.com/36028424/40570876-2175535e-60c2-11e8-8882-d8336ce488c7.png)
![hugo-coder-03](https://user-images.githubusercontent.com/36028424/40570877-23f0dbe4-60c2-11e8-8401-408866370af1.png)

**Comment by luizdepra** [comment]
Very good. I liked the color and everything.
Can we use the logo as a favicon too?

**Comment by Tobaloidee** [comment]
Yes Sir. I will send you the svg formats as well as the png..should i make a PR?

**Comment by luizdepra** [comment]
Sure. We can use the `image` to store the images. And add one or two into the README. :)

**Comment by Tobaloidee** [comment]
done with the PR #26 Sir @luizdepra 

Thanks so much for accepting and using the design! 

-tobaloidee

**Comment by Tobaloidee** [comment]
Hi Sir @luizdepra , you can download the other files (monochrome versions) on this [link](https://drive.google.com/drive/u/0/folders/1Pb1nmOMfLCaTdYwn1F4Xn3ATthHawFUK) to my drive. Thanks again!

**Comment by luizdepra** [comment]
Nice! I think I'll change to the black version. It suits better.

**Comment by Tobaloidee** [comment]
It's all yours Sir! Thanks again and best regards!! Thank you for accepting my proposal Sir..
